### PR TITLE
Get Possible Next states doesn't work

### PR DESCRIPTION
**Pull request by Akira83**
Changes:
AnalysisInspector.js
	Added a new tab window with that shows the input json file when the
flag develop is true.

backendComm.js 
	Added the localhost execution when the develop flag is true. Corrected
the flags values for get all possible next states and find single
solution when calling backend results 
	
main.js
	Added the flag develop. If set as true it will show all console logs
and the window with input file to backend.
	
SolveModel.java
	Just added a way to the output from execution to be pretty.

### CODE REVIEW

**Comment by Akira83** [comment]
In the test that I executed the output model generate duplicated values for all nodes in the model. I don't know if this is correct.

**Comment by amgrubb** [comment]
Marcel, you need to create a pull request to merge into `develop` not `master`.

**Comment by amgrubb** [comment]
Can you undo your commit: `Merge branch 'master' into feature-get-possible-next-states`?

File: leaf-ui/js/backendComm.js [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
..."]},{"id":"1","status":["0011","0011"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["0010","0010"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["1110","1110"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["1100","1100"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["1100","1100"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["1110","1110"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0110","0110"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0111","0111"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0100","0100"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0010","0010"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0011","0011"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0000","0000"]}]}],"finalAssignedEpoch":[],"finalValueTimePoints":[],"relativeTimePoints":5,"absoluteTimePoints":[1,3]};
```
Comment (amgrubb) [inline leaf-ui/js/backendComm.js]: Why are you committing hard coded values??

File: leaf-ui/js/backendComm.js [hunk 2]
```diff
@@ -67,10 +71,9 @@ function getFileResults(isGetNextSteps){
 		success: function(response){
 			analysisResults = JSON.parse(response);
 			if (analysisResults == ""){
-				alert("Ops! We couldn't read output.out file.")
+				alert("Error while reading the resonse file from server. This can be due an error in executing java application.")
```
Comment (amgrubb) [inline leaf-ui/js/backendComm.js]: I like your updated error messages. :)

File: leaf-ui/js/backendComm.js (hunk 1 above)
Comment (Akira83) [reply leaf-ui/js/backendComm.js]: This is just an example if you want to execute in localhost.
But I can remove, no problem.

**Comment by Akira83** [comment]
Ops... I didn't notice it was in master and not develop....

**Pull request closed without merging**